src/pages/Main/Main.js

```diff
@@ -1,5 +1,66 @@
+import { useState, useEffect } from 'react';
+
+import Banner from './Banner/Banner';
+import Product from './Product/Product';
+
+import './Main.scss';
```

Comment:
import 순서에 따라 잘 정렬해주셨고, 줄바꿈만 없애주세요!!
```suggestion
import { useState, useEffect } from 'react';
import Banner from './Banner/Banner';
import Product from './Product/Product';
import './Main.scss';
```